src/components/shared-panel/shared-panel.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import {
+  EmailShareButton,
+  FacebookMessengerShareButton,
+  FacebookShareButton,
+  TwitterShareButton,
+  WhatsappShareButton,
+} from 'react-share';
+import styled from 'styled-components';
+import { up } from '../breakpoint/breakpoint';
+import { IconEmail } from '../icons/social/email';
+import { IconFacebook } from '../icons/social/facebook';
+import { IconMessenger } from '../icons/social/messenger';
+import { IconTwitter } from '../icons/social/twitter';
+import { IconWhatsapp } from '../icons/social/whatsapp';
+
+export interface AuthorCredentialsProps {
```

Comment:
As this interface defines the props of the SharedPanel, it should be called `SharedPanelProps`

Reply:
As far as we don't need facebookId, we can remove it completely 🌻